model/components/prometheus.yml

```diff
@@ -0,0 +1,95 @@
+---
+components:
+
+  # prometheus ClientLibs
+  - id: prometheus-client-libs
+    name: Prometheus ClientLibs
+    description: Prometheus client-libraries for instrumenting application code
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - agent
+    capabilities:
+      - tech:
+        - java
+        - go
+        - scala
+        - python
+        - ruby
+        - bash
+        - c++
+        - common-lisp
+        - elixir
+        - erlang
+        - haskell
+        - lua
+        - .net
+        - c#
+        - nodejs
+        - php
+        - rust
+      - aspects:
+        - metrics-app
+        - metrics-process
+
+  # prometheus Server
+  - id: prometheus-server
+    name: Prometheus Server
+    description: Prometheus server is responsible for scraping and storing time series data collected from the client libraries and exporters
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - collector
+      - visualization
+      - storage
+    connections:
+      - graphite
+      - influx-db
+      - chronix
+      - weavecortex
+      - crate-db
+      - open-tsdb
+      - postgresql
+      - timescala-db
+      - grafana
+      - prometheus-exporter
+      - prometheus
```

Comment:
this should be now changed to `prometheus-client-libs` right?

Reply:
yes right forgot about this one ^^